Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -302,7 +302,19 @@ public void onResult(@Nullable VenmoAccountNonce venmoAccountNonce, @Nullable Ex
             if (venmoResult.getError() instanceof UserCanceledException) {
                 braintreeClient.sendAnalyticsEvent("pay-with-venmo.app-switch.canceled");
             }
-            listener.onVenmoFailure(venmoResult.getError());
+            deliverVenmoFailure(venmoResult.getError());
+        }
+    }
+
+    private void deliverVenmoSuccess(VenmoAccountNonce venmoAccountNonce) {
+        if (listener != null) {
```

Comment:
Do we want to handle if the listener is null in either of these cases? Not sure of the best way to do that or even if that'd be helpful to a merchant.

Reply:
I'm not really sure there is any way to handle when the listener is null aside from just not crashing - without the listener we can't deliver a result, and updating the method to throw an error would be a breaking change.

Reply:
🤔 is there any way for us to output something to the console? Mainly thinking for merchants who do something weird to have anything to point them potentially towards what they may have done wrong.

Reply:
Added warning logs here if the listeners are null